src/index.ts

```diff
@@ -642,8 +642,10 @@ const formats: IFormat[] = [
   getConfig('SOL', 501, bs58Encode, bs58Decode),
   hexChecksumChain('XDAI', 700),
   hexChecksumChain('VET', 703),
+  bech32Chain('AVA', 645, 'ava'),
   bech32Chain('BNB', 714, 'bnb'),
   getConfig('HIVE', 825, steemAddressEncoder, steemAddressDecoder),
+  hexChecksumChain('TOMO', 889),
```

Comment:
Is this correct? Your reference https://docs.tomochain.com/developer-guide/integration/tomop-exchange-integration says it's base58 of `public view key (33 bytes) | public spend key (33 bytes) | checksum (4 bytes)`